mlrose/opt_probs.py

```diff
@@ -275,13 +275,69 @@ def __init__(self, length, fitness_fn, maximize=True, max_val=2):
     def eval_node_probs(self):
         """Update probability density estimates.
         """
-        # Create mutual info matrix
-        mutual_info = np.zeros([self.length, self.length])
-        for i in range(self.length - 1):
-            for j in range(i + 1, self.length):
-                mutual_info[i, j] = -1 * mutual_info_score(
-                    self.keep_sample[:, i],
-                    self.keep_sample[:, j])
+        if (self.mimic_speed == False):
```

Comment:
`self.mimic_speed` hasn't previously been defined for this Class, so this line is causing errors when I run the unit tests on this code.

Reply:
Initialized self.mimic_speed = False in DiscreteOpt base class __init__ method